system/jlib/jfile.cpp

```diff
@@ -760,6 +760,69 @@ bool CFile::remove()
 #endif
 }
 
+static std::atomic<unsigned> renameRetries{(unsigned)-1}; // initially uninitialized, set on 1st use (in rename)
+
+void setRenameRetries(unsigned numRetries)
+{
+    renameRetries = numRetries;
+}
+
+static void doRename(const char *src, const char *dst, const char *errMsg)
+{
+    unsigned retriesLeft = renameRetries;
+    unsigned retry = 0;
+    while (true)
+    {
+        if ((unsigned)-1 == retriesLeft)
```

Comment:
Would be a bit cleaner if this was in a getRenameRetries() function which was called to initialise retriesLeft at the start of the function.

Reply:
the rename should have been above this code (I move it in some testing and forgot to move it back),
such that it only when through the renameRetries initialization / read config code on 1st rename failure.
I didn't want it go through config lookup in the std. (successful) case.

I'll change.